### PR TITLE
More complicated tag helper example

### PR DESCRIPTION
**Pull request by jherdman**
### Motivation / Background

In using the [`tag` helper](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag) I found myself a little confused about how to use the block form of the helper with attributes. I wanted to ensure this usage was captured in the documentation for others.

### Detail

I've changed the example tag and added some attributes to illustrate usage.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by p8** [comment]
Thanks for the PR @jherdman 
The new example is a bit more complex.
I think it might help here if we keep the old simple example and add the new example under it.

**Comment by jherdman** [comment]
@p8 done!

File: actionview/lib/action_view/helpers/tag_helper.rb [hunk 1]
```diff
@@ -420,6 +420,14 @@ def method_missing(called, *args, escape: true, **options, &block)
       #   # A void element:
       #   tag.br  # => <br>
       #
+      # Note that when using the block form you'll need to wrap options in
+      # parentheses.
```
Comment (p8) [inline actionview/lib/action_view/helpers/tag_helper.rb:424]: We try to [avoid](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns) using "you" in the API docs.
Maybe something like:
```suggestion
      # Note that when using the block form options should be wrapped
      # in parentheses.
```

**Review by p8 — CHANGES_REQUESTED** [review]

File: actionview/lib/action_view/helpers/tag_helper.rb (hunk 1 above)
Comment (jherdman) [reply actionview/lib/action_view/helpers/tag_helper.rb]: Done!

**Comment by p8** [comment]
Thanks @jherdman

**Pull request merged**